src/core/pagination/relay.style.pagination.fn.ts

```diff
@@ -75,19 +82,55 @@ export const getRelayStylePaginationResults = async <
   const hasWhere =
     query.expressionMap.wheres && query.expressionMap.wheres.length > 0;
 
+  let afterRowIdResult: T | undefined = undefined;
```

Comment:
I believe there isn't a use case to apply both cursors together. So those conditions below are redundant.